Client/DebugMode.cpp

```diff
@@ -0,0 +1,30 @@
+#include "DebugMode.h"
+
+void NCL::CSC8503::DebugMode::DisplayDebug(float dt)
+{
+	GetFPSCount(dt);
+	GetMemoryUsage();
+}
+
+void NCL::CSC8503::DebugMode::GetFPSCount(float dt)
```

Comment:
"Get" suggests that the function will return the FPS count: this should probably be "DisplayFPSCount" instead, likewise for GetMemoryUsage

Reply:
Agreed